### PR TITLE
gamemode: improve colors, some refactors and add alias for some commands

### PR DESCRIPTION
**Pull request by duboiss**

### CODE REVIEW

File: SemiRP/Utils/Chat.cs [hunk 1]
```diff
@@ -38,60 +36,75 @@ public static void SendGradientRangedChat(Player from, float radius, Color baseC
...
         public static void SendDoChat(Player from, string message)
         {
-            SendRangedChat(from, SemiRP.Constants.PROXIMITY_RADIUS, Constants.Chat.ME, "* " + message + " ((" + from.Name + "))");
+            SendRangedChat(from, SemiRP.Constants.PROXIMITY_RADIUS, Constants.Chat.ME, message + " ((" + from.Name + "))");
         }
+
         public static void ErrorChat(Player player, string message)
         {
-            player.SendClientMessage(Color.White, "[" + Constants.Chat.ERROR_TAG + "ERREUR" + Color.White + "] "+message);
+            player.SendClientMessage(Constants.Chat.ERROR, message);
```
Comment (Papawy) [inline SemiRP/Utils/Chat.cs:59]: No more "ERROR" tag in the chat ?

**Review by Papawy — CHANGES_REQUESTED** [review]
It's nice, good job. The only thing is the "ERROR" tag in ErrorChat that was remvoed: I am not really convinced by this modification.

File: SemiRP/Utils/Chat.cs (hunk 1 above)
Comment (duboiss) [reply SemiRP/Utils/Chat.cs:59]: Effectively. It is redundant to have a full chat with line-by-line "tags".

This is not a problem from a user point of view:
- he knows his action didn't go as planned
- he always see the colored error

**Review by Papawy — APPROVED** [review]

File: SemiRP/Commands/ChatCommands.cs [hunk 2]
```diff
@@ -12,57 +9,57 @@ class ChatCommands
...
             if (!receiver.AcceptMP)
             {
-                Utils.Chat.ErrorChat(sender, receiver.Name + " n'accepte pas de MP.");
+                Chat.ErrorChat(sender, receiver.Name + " n'accepte pas de MP.");
                 return;
             }
 
-            sender.SendClientMessage(Constants.Chat.PM, "[MP] " + sender.Name + " [" + sender.Id + "] : " + message);
-            receiver.SendClientMessage(Constants.Chat.PM, "[MP] " + sender.Name + " [" + sender.Id + "] : " + message);
+            sender.SendClientMessage(Color.Yellow, "[MP] Envoyé à " + sender.Name + "(" + sender.Id + ") : " + message);
```
Comment (V1ck3s) [inline SemiRP/Commands/ChatCommands.cs:61]: Why remove constants use and replace it with hardcoded color ?

File: SemiRP/Commands/VehicleCommands.cs [hunk 3]
```diff
@@ -129,60 +122,58 @@ private static void Hood(Player sender)
...
-                Utils.Chat.SendMeChat(sender, "allume le moteur de son véhicule.");
+                Chat.SendMeChat(sender, "allume le moteur de son véhicule.");
             vehicle.Engine = !vehicle.Engine;
         }
 
         [Command("liste", "list")]
         private static void List(Player sender)
         {
-            if (!Vehicle.All.Any(v => ((Vehicle)v).Data.Owner == sender.ActiveCharacter))
+            if (Vehicle.All.All(v => ((Vehicle) v).Data.Owner != sender.ActiveCharacter))
```
Comment (V1ck3s) [inline SemiRP/Commands/VehicleCommands.cs:167]: Check if it gives the right result

File: SemiRP/Utils/Vehicles/CmdHelper.cs [hunk 4]
```diff
@@ -23,47 +22,56 @@ public static Vehicle GetCurrentVehicleOrID(Player player, int id = -1)
 
             if (id != -1)
             {
-                if (!Vehicle.All.Any(v => v.Id == id))
+                if (Vehicle.All.All(v => v.Id != id))
```
Comment (V1ck3s) [inline SemiRP/Utils/Vehicles/CmdHelper.cs:25]: Check if it gives the right result

File: SemiRP/Commands/VehicleCommands.cs [hunk 5]
```diff
@@ -17,61 +17,56 @@ public class VehicleCommands
...
-                vehicle.Data.SpawnLocation.Position = new SampSharp.GameMode.Vector3(vehicle.Position.X, vehicle.Position.Y, vehicle.Position.Z + 0.5f);
+                Chat.InfoChat(sender, "Vous avez garé votre véhicule, il réapparaitra désormais ici.");
+                vehicle.Data.SpawnLocation.Position =
+                    new Vector3(vehicle.Position.X, vehicle.Position.Y, vehicle.Position.Z + 0.5f);
                 vehicle.Data.SpawnLocation.RotZ = vehicle.Angle;
 
                 var vData = vehicle.Data;
 
-                ServerDbContext dbContext = ((GameMode)GameMode.Instance).DbContext;
+                ServerDbContext dbContext = ((GameMode) BaseMode.Instance).DbContext;
```
Comment (V1ck3s) [inline SemiRP/Commands/VehicleCommands.cs:62]: Why change GameMode to BaseMode ?

**Review by V1ck3s — COMMENTED** [review]
Some modifications that remain to be discussed, for the rest it's good.

**Comment by V1ck3s** [comment]
Fixed in 6c1311608ec3a552ec2103d9982f510cae2b27a8

**Pull request merged**